File: app/src/main/java/com/example/anxiao/mytestapplication/app/Logger.java
package com.example.anxiao.mytestapplication.app;

import android.util.Log;


public class Logger {

    public static void debug(Object log) {
        Log.d("anmnight", String.valueOf(log));
    }

    public static void info(Object log) {
        Log.i("anmnight", String.valueOf(log));
    }

    public static void err(Object log) {
        Log.e("anmnight", String.valueOf(log));
    }

    public static void err(Class clz, Object log) {
        Log.e(clz.getName(), String.valueOf(log));
    }

}

File: app/src/main/java/unit/Logger.kt
package unit

import android.util.Log

/**
 * https://github.com/anmnight
 * author：anxiao on 2018/8/3 15:07
 * anmnight@qq.com
 */
object Logger {

    private const val TAG = "TestApplication"

    fun debug(any: Any) {
        Log.d(TAG, any.toString())
    }

    fun info(any: Any) {
        Log.i(TAG, any.toString())
    }

    fun error(any: Any) {
        Log.e(TAG, any.toString())
    }
}